### DISCUSSION TITLE
Release 1.2.4

### DISCUSSION THREAD

**Discussion by ThomasWaldmann** [borgbackup/borg#7472] [General]
## borgbackup 1.2.4 release

Short changelog summary (changes since borg 1.1):

https://www.borgbackup.org/releases/borg-1.2.html

Long changelog (including upgrade and compatibility notes):

https://github.com/borgbackup/borg/blob/1.2.4/docs/changes.rst#version-124-2023-03-24

### Installation

If you use pip to install this, use: pip install pkgconfig ; pip install "borgbackup==1.2.4"

For other installation methods and more details, please see: https://borgbackup.org/

### Feedback

<hr /><em>This discussion was created from the release <a href='https://github.com/borgbackup/borg/releases/tag/1.2.4'>Release 1.2.4</a>.</em>

**Comment by RichardTuma** [comment]
Is it compatible with ppc64 linux? Thx. Richard

**Reply by ThomasWaldmann** [reply]
@RichardTuma I don't know anything telling the opposite, but you might be the best person to actually try it.

In general, testing on big-endian architectures has become rather hard, because they are rare (and also OSes supporting them).
But recently there was some feedback from S390 testing and the minor testing issues were **not** BE related.

Also, Debian builds / tests on quite some archs: https://buildd.debian.org/status/package.php?p=borgbackup

**Comment by RichardTuma** [comment]
Thanks for a response. 
  
I am testing it now on little-endian, AlmaLinux 9.1 on IBM Power9 and it seems to be running well...  
   
  
Kind Regards, 
  
Richard Tůma

**Comment by ThomasWaldmann** [comment]
@RichardTuma Yeah, LE should be rather unproblematic as most people use borg on LE archs.

Note: I edited your public github discussions post and removed your email footer for better privacy.

**Comment by dragetd** [comment]
I somehow find the changelogs always a bit confusing. Am I the only one?

Currently I see:

* Short changelog -> Links to a summary that does not have any info about the current release.
* Long changelog -> Links to the changelog where you have to skip over 14 lines before seeing the actual new changes.

My suggestions:

* Dropthe notes before each changelog about the migration notes, or shorten them to a single sentence like 'If you are updating from before Borg 1.2, make sure to read ...link to dedicated migration notes...
* On the Release, rename 'Short changelog' to 'Borg 1.1 to 1.2 migrations notes' or 'Borg 1.2 changes summary', but something without 'changelog' in it.
* Below that, copy the latest snippet from the changelog with the changes for the current version.


That could be done manually or, with some more effort, be cobbled together with tools like 'semantic_releases'. Tho if the releases are made manually right now anyways, things could be kept simple (that way).

Just a suggestion. :-)

**Comment by ThomasWaldmann** [comment]
I'll try to improve it for the next release.

The migration notes are intentionally kept at the top, so users will see and hopefully read them.

I don't use automated tools to make the changelog. Not all commit comments should end up in the changelog and even some commit comments which should need rephrasing or more info for better clarity.

**Comment by Sunpresent** [comment]
Hi, I updated brog from version 1.1.16 to version 1.2.4, but I noticed that my backups are filling up faster than before. In other words, the new version takes more space than the previous version, considering that the rate of data space increase is almost constant
 have been. In one particular case, which seems very strange, 7 TB of space was taken from a 1 TB disk that was backed up twice, which is very, very strange. I would be very grateful if you could advise me if this is a bug or if I should change the way I use it.

**Reply by ThomasWaldmann** [reply]
Guess you missed the note in the change log that now `borg compact` is (additionally) required to free space.

**Reply by Sunpresent** [reply]
Thank you for your response, yes my problem is solved
